core/.js/src/test/scala/io/udash/bindings/TagsBindingTest.scala

```diff
@@ -1040,12 +1070,12 @@ class TagsBindingTest extends UdashFrontendTest with Bindings { bindings: Bindin
       val p = SeqProperty[String]("a", "b", "c")
       val template = div(
         span(),
-        produceWithNested(p, customElementsReplace = customReplace(true))((v, _) => div(v.mkString(",")).render),
+        produce(p, customElementsReplace = customReplace(true))(v => div(v.mkString(",")).render),
         span()
       ).render
       val template2 = div(
         span(),
-        produceWithNested(p, customElementsReplace = customReplace(false))((v, _) => div(v.mkString(",")).render),
+        produce(p, customElementsReplace = customReplace(false))(v => div(v.mkString(",")).render),
```

Comment:
So, `produceWithNested` can now skip the argument? Improve coverage, not maintain it.